### PR TITLE
New recipe addressing #549

### LINKED ISSUE #549: add recipe to cut hypergraph to a max_order
I thought we had this somewhere in tutorials or recipes but I cannot find it any more

### PR DESCRIPTION
**Pull request by nwlandry**
This fixes #549.

### CODE REVIEW

**Comment by maximelucas** [comment]
Two small comments but looks good!

**Comment by nwlandry** [comment]
I think I addressed all of your comments, @maximelucas!

File: tutorials/recipes/recipes.ipynb [hunk 1]
```diff
@@ -87,7 +87,7 @@
    "outputs": [
```
[the remark this answers is not in the document]
Comment (nwlandry) [reply tutorials/recipes/recipes.ipynb:87]: Good catch. I added an additional line with this keyword to illustrate.

[the remark this answers is not in the document]
Comment (nwlandry) [reply tutorials/recipes/recipes.ipynb:87]: Good catch! The issue was in the relabel function, and it is fixed now with unit tests as well.

**Review by maximelucas — APPROVED** [review]

**Pull request merged**